### DISCUSSION TITLE
Show the navigation bar regardless of the zoom on your browser

### DISCUSSION THREAD

**Discussion by sierratull** [GitbookIO/community#89] [Feature Requests]
### What's your feature/idea?

When zoomed in on your browser and editing a page, the contains of the side bar navigation no longer populates on the page

### What's your desired outcome?

Have the ability to zoom in on any browser and continue to the see the side bar navigation for a page

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by taszap** [comment]
#232 Related, closing in favour of the latest request
